src/make-plural.js

```diff
@@ -24,6 +24,7 @@ class Parser {
       return 'n == 1 && v0'
     }
     return cond
+      .replace(/([^=\s])([!=%]+)([^=\s])/g, '$1 $2 $3')
```

Comment:
As I went and refactored everything with #14, it would be good if you could rebase this PR to just this line.